packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -1304,22 +474,16 @@ const AccountSwitcher = props => {
                 </Text>
             </div>
             <Text color='less-prominent' line_height='xs' size='xxxs' className='acc-switcher__total-subtitle'>
-                {total_assets_message}
+                {localize('Total assets in your Deriv accounts.')}
             </Text>
             <div className='acc-switcher__separator' />
 
             <TradersHubRedirect />
 
-            <div
-                className={classNames('acc-switcher__footer', {
-                    'acc-switcher__footer--traders-hub': props.is_pre_appstore,
-                })}
-            >
-                {props.is_pre_appstore && isRealAccountTab && props.has_active_real_account && !props.is_virtual && (
+            <div className='acc-switcher__footer'>
```

Comment:
`acc-switcher__footer--traders-hub` is missed in className

Reply:
We dont need the traders hub class. Ill remove it. It gets messed up if i use that